faq.md

```diff
@@ -33,6 +33,9 @@ This URL can be accessed directly by CLI utilities like `curl` and `wget`, but i
     javascript:location.href='https://clients2.google.com/service/update2/crx?response=redirect&acceptformat=crx2,crx3&prodversion='+(navigator.appVersion.match(/Chrome\/(\S+)/)[1])+'&x=id%'+'3D'+(document.querySelector('a[href^="https://chrome.google.com/webstore/report/"]').pathname.match(/[^\/]+\/*$/)[0])+'%'+'26installsource%'+'3Dondemand%'+'26uc';
     ```
 
+*  Third-party CRX downloaders
+Another option can be using a third-party CRX downloader available on the web. You'll just need to paste the extension URL from the chrome webstore.
```

Comment:
Is it necessary to mention this method when the Bookmarklet and the built-in features have similar convenience? If it is more convenient, could you link to the specific extensions?

Also, please check your Markdown formatting.